client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/multimap/RemoteMultimapCacheImpl.java

```diff
@@ -36,7 +37,7 @@
    private static final Log log = LogFactory.getLog(RemoteMultimapCacheImpl.class, Log.class);
    private static final boolean trace = log.isTraceEnabled();
 
-   private RemoteCacheImpl<K, Collection<V>> cache;
+   private InternalRemoteCache<K, Collection<V>> cache;
```

Comment:
```suggestion
   private final InternalRemoteCache<K, Collection<V>> cache;
```

Reply:
offtopic: why do we need lifespan and max idle?
```
   private long defaultLifespan = 0;
   private long defaultMaxIdleTime = 0;
```
They never change and they aren't set anywhere
cc @karesti